### ISSUE TITLE
Docker Image Published without Version Tag

### ISSUE THREAD

**Issue by brendongo** [localstack/localstack#1169]
See https://hub.docker.com/r/localstack/localstack/tags

A latest tag was published 16 hours ago without an explicit semver. Was this intended?

**Comment by brendongo** [comment]
There also does not seem to be a release tag on github or a PR intending a release

**Comment by brendongo** [comment]
I also get the following Traceback when using the latest docker container and making calls to S3:

```
localstack_1  | 2019-03-04T20:00:18:WARNING:localstack.services.generic_proxy: Connection prematurely closed by client (broken pipe).
localstack_1  | Traceback (most recent call last):
localstack_1  |   File "/usr/lib/python2.7/SocketServer.py", line 596, in process_request_thread
localstack_1  |     self.finish_request(request, client_address)
localstack_1  |   File "/usr/lib/python2.7/SocketServer.py", line 331, in finish_request
localstack_1  |     self.RequestHandlerClass(request, client_address, self)
localstack_1  |   File "/opt/code/localstack/localstack/services/generic_proxy.py", line 81, in __init__
localstack_1  |     BaseHTTPRequestHandler.__init__(self, request, client_address, server)
localstack_1  |   File "/usr/lib/python2.7/SocketServer.py", line 654, in __init__
localstack_1  |     self.finish()
localstack_1  |   File "/usr/lib/python2.7/SocketServer.py", line 713, in finish
localstack_1  |     self.wfile.close()
localstack_1  |   File "/usr/lib/python2.7/socket.py", line 283, in close
localstack_1  |     self.flush()
localstack_1  |   File "/usr/lib/python2.7/socket.py", line 307, in flush
localstack_1  |     self._sock.sendall(view[write_offset:write_offset+buffer_size])
localstack_1  | error: [Errno 32] Broken pipe
```

In particular this was in a call to `boto3.resource("s3").Bucket(BUCKET_NAME).download_file(KEY, DEST)`

Solved the issue by pinning to 0.9.0

**Comment by whummer** [comment]
Please do not rely on the `latest` version of the image being fully functional. The `latest` version gets pushed by PR builds on `master`, and, occasionally, for testing certain features.

Please continue pinning your image to a release tag if you want to work with a stable version. Thanks
